documentation/docs/ui/attachment_gallery.md

```diff
@@ -0,0 +1,45 @@
+---
+id: uiAttachmentGallery
+title: Attachment Gallery
+sidebar_position: 6
+---
+
+_AttachmentGalleryActivity_ is an _Activity_ used to display attachments that the users have sent in the chat. It is an image gallery where users can see the pictures, share, download, and use a menu to navigate through the pictures.
```

Comment:
Do we use italic of code style for works like AttachmentGalleryActivity?

Reply:
Good question. We should be consistent across all documents. I'll add this to the next weekly agenda.